cognito-pre-token-generation/src/test/java/no/unit/nva/cognito/TriggerHandlerTest.java

```diff
@@ -81,10 +81,12 @@
 
     public static final String CREATOR = "Creator";
     public static final String USER = "User";
-    public static final String SAMPLE_CRISTIN_ID = "http://cristin.id";
+
     public static final AdminUpdateUserAttributesResult UNUSED_RESULT = null;
     public static final Javers JAVERS = JaversBuilder.javers().build();
     public static final Context CONTEXT = mock(Context.class);
+    public static final URI SAMPLE_CRISTIN_ORG_ID = randomCristinOrgId();
```

Comment:
Create a valid CristinId instead of an arbitrary one. A valid Cristin ID is a  URI that has the same host as the identity service and the path starts with /cristin/organization